assets/js/search-utilities.js

```diff
@@ -187,31 +188,63 @@ function constructTitleSuggestions(entries, resultTitlesIds, suggestionDescripti
   }
 }
 
+function constructTitle(storedEntry) {
+  const title = document.createElement('div');
+  title.textContent = storedEntry.title;
+  title.classList.add('suggestion__title');
+
+  return title;
+}
+
+function constructAnchor(title, storedEntry) {
+  const a = document.createElement('a');
+  a.href = storedEntry.href;
+  a.appendChild(title);
+
+  return a;
+}
+
+function constructFullSearchLengthInfo(allMatches) {
+  const fullResultsLengthInfo = document.createElement('span');
+  fullResultsLengthInfo.style['font-size'] = '13px';
+  fullResultsLengthInfo.style.display = 'inline';
+  fullResultsLengthInfo.innerHTML = `[${allMatches.length} more occurrences]`;
+
+  return fullResultsLengthInfo;
+}
+
+function constructDescription(descriptionText, fullResultsLengthInfo, suggestionDescriptionClass) {
+  const description = document.createElement('div');
+  description.innerHTML = descriptionText;
+  description.appendChild(fullResultsLengthInfo);
+  description.classList.add(suggestionDescriptionClass);
+
+  return description;
+}
+
 function constructContentSuggestions(entries, searchQuery, resultIds, suggestionDescriptionClass, isLiveSearch = false, resultsLimit) {
   for (const id of resultIds) {
     if (isLiveSearch && entries.length === MAX_CONTENT_SUGGESTION_ELEMENTS) break;
     const storedEntry = index.get(id);
     const startPositions = getMatchIndices(storedEntry.content.toLowerCase(), searchQuery.toLowerCase(), resultsLimit);
+    const allMatches = getMatchIndices(storedEntry.content.toLowerCase(), searchQuery.toLowerCase());
 
-    for (const startPosition of startPositions) {
-      const entry = document.createElement('div');
-      const a = document.createElement('a');
-      a.href = storedEntry.href;
-      entry.appendChild(a);
-
-      const title = document.createElement('div');
-      title.textContent = storedEntry.title;
-      title.classList.add('suggestion__title');
-      a.appendChild(title);
-
-      const description = document.createElement('div');
-      description.innerHTML = getFoundWordFromDescription(storedEntry.content, searchQuery, startPosition);
-      description.classList.add(suggestionDescriptionClass);
-      a.appendChild(description);
+    const title = constructTitle(storedEntry);
+    const a = constructAnchor(title, storedEntry);
+    const entry = document.createElement('div');
+    let descriptionText = '';
 
+    for (const startPosition of startPositions) {
+      descriptionText += getFoundWordFromDescription(storedEntry.content, searchQuery, startPosition) + '...';
```

Comment:
```suggestion
      descriptionText += getFoundWordFromDescription(storedEntry.content, searchQuery, startPosition) + '... ';
```

to prevent '...[ 50 occurrences ]' and instead to be '... [ 50 occurrences ]'

Reply:
Fixed.